x-pack/plugins/snapshot_restore/public/application/sections/home/policy_list/policy_list.tsx

```diff
@@ -157,9 +165,36 @@ export const PolicyList: React.FunctionComponent<RouteComponentProps<MatchParams
     const policySchedules = policies.map((policy: SlmPolicy) => policy.schedule);
     const hasDuplicateSchedules = policySchedules.length > new Set(policySchedules).size;
     const hasRetention = Boolean(policies.find((policy: SlmPolicy) => policy.retention));
+    const isSlmRunning = slmStatus?.operation_mode === SLM_STATE.RUNNING;
 
     content = (
       <section data-test-subj="policyList">
+        {!isSlmRunning ? (
+          <Fragment>
+            <EuiCallOut
+              title="Snapshot lifecycle management (SLM) is not running"
+              color="warning"
+              iconType="warning"
+            >
+              <p>
+                {i18n.translate('xpack.snapshotRestore.slmWarning', {
+                  defaultMessage: 'Policies are not executed. You must restart SLM',
```

Comment:
The code might be clearer if we use `FormattedMessage` instead with a value for the link. See [this](https://github.com/elastic/kibana/blob/d1e5aa158248fef3a86a59be4212f0a8d511ff91/x-pack/plugins/index_management/public/application/sections/home/index_list/index_table/index_table.js#L591) for example.

Reply:
You are right, this is so much clearer. Fixed